.github/workflows/deploy.yml

```diff
@@ -82,6 +82,7 @@ jobs:
           REDIS_DB: 0
           TOKEN_METADATA_BUCKET: token-metadata-testnet
           CORS_ALLOWED_ORIGIN: https://explorer.testnet.hathor.network
+          NODE_CACHE_TTL: 5
```

Comment:
5 seconds? I feel we should wait at least 30 seconds before removing the data.

Reply:
Ok, we all agree with 30 then. Changed.